migrations/internal/m20211220/migration.go

```diff
@@ -16,7 +15,7 @@ const ID = "20211220"
 type Job struct {
 	ID                     uuid.UUID      `gorm:"column:id;primary_key;type:uuid;"`
 	Type                   string         `gorm:"column:type"`
-	State                  jobs.State     `gorm:"column:state;default:INIT"`
+	State                  string         `gorm:"column:state;default:INIT"`
```

Comment:
removed import of `jobs` package to avoid cyclic import

Reply:
Oh I didn't take into account that we are importing the migrations package in our datastore/gorm package. I will do a PR to fix this.

Reply:
Here is the PR: https://github.com/flow-hydraulics/flow-wallet-api/pull/264

Reply:
It is now merged in

Reply:
@latenssi Thank you. Updated PR.